### PR TITLE
feat(sig-services): Add secure data handling requirement

### LINKED ISSUE #109: [SIG Services][Guideline] Secure user data handling
Secure user data handling is defined in End user agreement/Ts&Cs 

Recording: https://drive.google.com/file/d/1eMIMtNLV48AuqFdH2RSckyWBhKSme9lf/view?usp=sharing

### PR DESCRIPTION
**Pull request by chauhankaranraj**
Adds a document describing the overview, requirements, and reasoning for the "secure data handling" rule discussed during [this](https://drive.google.com/file/d/1eMIMtNLV48AuqFdH2RSckyWBhKSme9lf/view?usp=sharing) SIG services meeting.

Closes #109

### CODE REVIEW

File: sig-services/service-requirements/secure-user-data-handling.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+This requirement states that the service should clearly outline how user data is securely handled for the duration of the contract, as well as after its termination.
+
+## Requirements
+
+- A file / page is created which describes user data handling in various contexts of the service, including the following
+  - how the data obtained during user sign-up is stored and used
+  - how the data obtained during service usage is stored and used
+  - what happens to user data after service contract terminates
+- If third party services have access to user data, then how they ensure secure data handling should also be outlined.
```
Comment (Shreyanand) [inline sig-services/service-requirements/secure-user-data-handling.md:15]:
```suggestion
- If third party services have access to user data, then how they ensure secure data handling should also be outlined.
- If the service complies with legal guidelines or regulations, for example the GDPR in EU, it should be explained. 
```

**Review by Shreyanand — COMMENTED** [review]
/lgtm, added a small point.

**Comment by chauhankaranraj** [comment]
> /lgtm, added a small point.

thanks, added the line :+1:

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**